static/lang/en.json

```diff
@@ -2409,7 +2409,7 @@
         "MaxStaminaTitle": "Your maximum stamina points",
         "Migrations": {
             "Finished": "PF2E System Migration to version {version} completed!",
-            "OnlyGMCanUse": "Only a gamemaster can use this tool.",
+            "OnlyGMCanUse": "Only a Gamemaster can use this tool.",
```

Comment:
```suggestion
            "OnlyGMCanUse": "Only a gamemaster can use this tool.",
```

Reply:
I thought it was always capitalized in text, but sure, it can stay like that